client/src/AddNewVideo.js

```diff
@@ -0,0 +1,98 @@
+
+import React, { useState } from "react";
+import './App.css';
+import axios from 'axios';
+
+
+const AddNewVideo = ({ addVideoButton }) => {
+const [title, setTitle] = useState('');
+const [url, setUrl] = useState('');
+// const [error, setError] = useState('');
+
+
+// const correctYoutubeUrl = (url) => {
+// try {
+// const youtubeUrl = new URL(url);
+// return (
+// youtubeUrl.hostname === 'www.youtube.com' ||
+// youtubeUrl.hostname === 'youtube.com'
+// );
+// } catch (error) {
+// return false;
+// }
+// };
+
+
+
+
+// const submitButton = async (e) => {
+// e.preventDefault();
+
+
+// if (title.trim() === '') {
+// setError('Title cannot be empty.');
+// return;
+// }
+
+
+// if (!correctYoutubeUrl(url)) {
+// setError('Invalid youTube URL.');
+// return;
+// }
+
+
+// setError('');
+const handleAddVideo = async () => {
+try {
```

Comment:
I'd generally recommend formatting your code with indentation, it'll make it much easier to read and follow, and to debug things like extra/missing brackets.